docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       - ${POSTGRES_PORT}:5432
     restart: always
     volumes:
-      - ${PWD}/docker-scripts:/docker-entrypoint-initdb.d
+      - ./docker-scripts:/docker-entrypoint-initdb.d
```

Comment:
The funny thing about this is that I don't think it works on Windows. Another cross-platform Docker issue. :) We can deal with that later though.

Reply:
^ about the line, not your change. It's been an issue since I added it. Just wanted to record the issue.

Reply:
I removed it because I am getting PWD variable undefined when running docker-compose